### DISCUSSION TITLE
IBAX testnet is online join airdrop now.

### DISCUSSION THREAD

**Discussion by HalloKyle** [IBAX-io/go-ibax#5] [Announcement]
You can earn IBXC coins by join IBAX Airdrop now.
Airdrop Entrance: https://ibax.io/login

**Comment by siriusjack19** [comment]
big project, so interesting

**Comment by imdatdalga** [comment]
very interesting project with big potential.

**Reply by Brightnetworks** [reply]
Good ambition for this project

**Comment by Brightnetworks** [comment]
Good ambition for this project
